sympy/functions/combinatorial/factorials.py

```diff
@@ -288,7 +288,35 @@ def _eval_as_leading_term(self, x):
         return self.func(arg)
 
 class MultiFactorial(CombinatorialFunction):
-    pass
+
+    @classmethod
+    def eval(cls, n, alpha):
+        if not alpha.is_integer or not alpha.is_positive:
+            raise ValueError("alpha should be a positive integer")
+        if alpha == 1:
+            return factorial(n)
+        if alpha == 2:
+            factorial2(n)
+
+        if n.is_Number:
+            if not n.is_Integer:
+                raise ValueError("argument should be non negative or negative"
```

Comment:
You should add a spacing at the end because it would show like ‘negativeinteger’